src/app/components/Menu/Menu.tsx

```diff
@@ -129,40 +156,59 @@ const Menu = () => {
                   {showLanguages && (
                     <div className="languagesListContainer">
                       <ul className="languagesList">
-                        <button
-                          value={'pl-PL'}
-                          type="button"
-                          className="languagesList__button"
-                          onClick={() => {
-                            handleLanguageSelection();
-                            setLanguage('pl-PL');
-                            // switchLocale('pl');
-                          }}
-                        >
-                          Polish
-                        </button>
-                        <button
-                          type="button"
-                          className="languagesList__button"
-                          onClick={() => {
-                            handleLanguageSelection();
-                            setLanguage('es-ES');
-                            // switchLocale('es');
-                          }}
-                        >
-                          Spanish
-                        </button>
-                        <button
-                          type="button"
-                          className="languagesList__button"
-                          onClick={() => {
-                            handleLanguageSelection();
-                            setLanguage('fr-FR');
-                            // switchLocale('fr');
-                          }}
-                        >
-                          French
-                        </button>
+                        {actualLanguage !== 'English' && (
```

Comment:
Zgadzam sie z tymonem. Pierdolnij tu jakiegos loopa bo przeciez to sa literalnie te same informacje. No i za kazdym razem jak bedziesz dodawał nowy jezyk to tez tutaj bedziesz musiał to wpisac.